yt/visualization/volume_rendering/input_events.py

```diff
@@ -3,11 +3,11 @@
 import random
 from collections import defaultdict, namedtuple
 
+import cyglfw3 as glfw
 import matplotlib.cm as cm
 import numpy as np
-
-import cyglfw3 as glfw
 import OpenGL.GL as GL
```

Comment:
might be worth adding OpenGL and cyglfw3 to the list of known third party in setup.cfg